implementations/rust/ockam/ockam_command/src/cluster/create.rs

```diff
@@ -39,6 +40,11 @@ pub struct CreateCommand {
     #[arg(long)]
     pub use_public_ecr: bool,
 
+    /// Whether to use the Docker cache when building the image.
+    /// It can be set using the `OCKAM_USE_DOCKER_CACHE` environment variable.
+    #[arg(long)]
```

Comment:
Could a standard clap environment variable work?
`#[arg(long, env = "OCKAM_...`

Reply:
I believe the limitation is that it's not parsed the same as we do with our function (e.g. parsing `1` or `true` as boolean).